cscs-checks/apps/greasy/greasy_check.py

```diff
@@ -161,58 +162,74 @@ def num_ranks(match):
                                 r'of \s*(\d+) from process \s*(\d+) out of '
                                 r'\s*(\d+)', output_file)
 
-            failure_msg = ('Found %s Hello, World... pattern(s), but expected '
-                           '%s pattern(s) inside the output file %s' % (
-                                              sn.count(result),
-                                              num_tasks * num_cpus_per_task,
-                                              output_file))
+            failure_msg = ('Found {0} Hello, World... pattern(s), but expected'
+                           ' {1} pattern(s) inside the output file {0}'.format(
+                               sn.count(result), num_tasks * num_cpus_per_task,
+                               output_file))
```

Comment:
f-strings, not `str.format()`. Check my suggestion above. This applies everywhere that you are using `format()`.